src/client.ts

```diff
@@ -117,6 +118,16 @@ export interface WhoIsResponse {
   stripe_id: string
 }
 
+export interface OrgInfo {
+  email: string
+  name: string
+  description: string
+  phone: string
+  metadata: { [key: string]: string }
+}
+
+export type LookupOrgResponse = WhoIsResponse & OrgInfo
```

Comment:
This seems less readable and more error prone than explicitly defining the fields. If you add a new field to either and they are not part of the actual response from the server then it could come as a surprise to users that some fields are not set. 

This could also be fixed by explicitly documenting which fields are set, but then you're typing in comments. 

Maybe I'm looking at this wrong?

Reply:
I mean, I could un-DRY it, like:

```ts
export interface OrgInfo {
  email: string
  name: string
  description: string
  phone: string
  metadata: { [key: string]: string }
}
export interface WhoIsResponse {
  org: OrgName
  stripe_id: string
}

// same effect as:
// export type LookupOrgResponse = WhoIsResponse & OrgInfo
export interface LookupOrgResponse {
  org: OrgName
  stripe_id: string
  email: string
  name: string
  description: string
  phone: string
  metadata: { [key: string]: string }
}
```

But as long as we expect to get the same set of data from `lookupOrg()` that we send as the `info` field to `/v1/subscribe`, seems fine to have them reference the same type.  If those things do deviate, it should be on purpose, and this makes it less likely that we let them drift by accident.

Reply:
> But as long as we expect to get the same set of data from lookupOrg() that we send as the info field to /v1/subscribe, seems fine to have them reference the same type.

They can drift. WhoIs may return more fields in OrgInfo than are supported by the subscribe endpoint. Nowhere does the API guarantee the two endpoints share a common understanding of what "org info" is. It just so happens to seem like they do, but that can change.